oct/cli/util/cloud_provider/instance_options.py

```diff
@@ -0,0 +1,20 @@
+# coding=utf-8
+from __future__ import absolute_import, division, print_function
+
+from click import option
+
+
+def instance_name_option(func):
```

Comment:
Is this used anywhere else?

Reply:
Let's un-func and re-func later if we need it.

Reply:
We don't want the func. fixed